.github/workflows/build.yml

```
@@ -2,30 +2,51 @@ name: "build"
 on:
   push:
   pull_request:
+  workflow_dispatch:
 jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [python27, python39, python310, python311]
-        docutils: [docutils016, docutils017, docutils018, docutils019, docutils020]
+        python: [python310, python311, python312]
```

I might also add 314, but we don't yet officially support it. Soon.

```suggestion
        python: [python310, python311, python312, python313]
```